openapi-v3.yaml

```diff
@@ -970,16 +970,25 @@ components:
       required:
         - account_id
         - project_id
+        - id
         - environment_id
         - name
+        - dbt_version
+        - triggers
         - execute_steps
+        - settings
+        - state
+        - schedule
       properties:
         account_id:
           type: "integer"
           example: 1
         project_id:
           type: "integer"
           example: 100
```

Comment:
Recommend adding a description:  "When modifying an existing job, project_id must match the project_id of the job being modified"

Reply:
I agree that this would be a nice addition, though right now with how its set up, we are defining the Job schema and reusing it for the request bodies on both the update and create actions in these docs. We would need to do a little refactoring I think so we could have different descriptions for the various fields and how they might different across the various api actions.

Reply:
I'd prefer to punt that to the next story/PR.